.gitignore

```diff
@@ -6,4 +6,7 @@ vendor
 *.bak
 *.out
 *.db
-*.env
\ No newline at end of file
+*.env
+*#
+*.#*
+*~
```

Comment:
what are these lines for?

Reply:
Emacs backup files

Reply:
these should be removed and added to your global .gitignore file